frontend/src/views/instructor/header.tsx

```diff
@@ -16,10 +16,8 @@ import {
 export function InstructorHeader() {
     const positions = useSelector(positionsSelector);
     const routes = positions.map((position) => ({
-        route: `/${position.id}`,
-        name:
-            `${position.position_code} ${position.position_title}` ||
-            `Position #${position.id}`,
+        route: `/positions/${position.id}`,
+        name: `${position.position_code}`,
```

Comment:
My Typescript (a newer one because I still can't switch back to the one we're using) complained position title can be `null`, this helps with that as I suppose the code is always defined? In our types it's still a `string` so it can technically be empty, do we still want a fallback?

Reply:
The title doesn't matter to the instructor. They know what course they're teaching :-). And the position code is always non-null